<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Edit_Light</class>
 <widget class="QDialog" name="Edit_Light">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Light</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Light Cone Angle</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="Light_Cone_Angle">
         <property name="maximum">
          <number>90</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Light Intensity</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="Light_Intensity">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLineEdit" name="LightName"/>
       </item>
       <item>
        <widget class="QLineEdit" name="Test"/>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber">
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>64</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="Change_Light_Color">
       <property name="text">
        <string>Change Light Color</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Light Origin</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>X-Coordinate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="Light_X_Coordinate_Origin">
           <property name="minimum">
            <number>-999</number>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
           <property name="value">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Y-Coordinate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="Light_Y_Coordinate_Origin">
           <property name="minimum">
            <number>-999</number>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
           <property name="value">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Z-Coordinate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="Light_Z_Coordinate_Origin">
           <property name="minimum">
            <number>-999</number>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
           <property name="value">
            <number>15</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Light Focal Point</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>X-Coordinate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="Light_X_Coordinate_Focal_Point">
           <property name="minimum">
            <number>-999</number>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Y-Coordinate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="Light_Y_Coordinate_Focal_Point">
           <property name="minimum">
            <number>-999</number>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Z-Coordinate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="Light_Z_Coordinate_Focal_Point">
           <property name="minimum">
            <number>-999</number>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="Light_Switch">
     <property name="text">
      <string>Light Switch</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>Change_Light_Color</tabstop>
  <tabstop>Light_X_Coordinate_Origin</tabstop>
  <tabstop>Light_Y_Coordinate_Origin</tabstop>
  <tabstop>Light_Z_Coordinate_Origin</tabstop>
  <tabstop>Light_X_Coordinate_Focal_Point</tabstop>
  <tabstop>Light_Y_Coordinate_Focal_Point</tabstop>
  <tabstop>Light_Z_Coordinate_Focal_Point</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Edit_Light</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Edit_Light</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>307</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Light_Cone_Angle</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Light_Intensity</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
